docs/modules/ROOT/pages/running-commodore.adoc

```diff
@@ -45,7 +45,47 @@ commodore() {
 == macOS X
 
 On macOS with Docker for Mac mounting the SSH agents socket into a container doesn't work.
-Instead your SSH key needs to be mounted into the container.
+Instead you need to mount as magic path that's provided by Docker for Mac.
+
+[NOTE]
+====
+This only works for the stock ssh-agent coming along with macOS.
+If you use any other agent, you are out of luck.
```

Comment:
Should we mention that other agents work as well, as long as they allow to store their socket file in a certain directory?

Reply:
Do you have a working example? If not, no as it is not confirmed knowledge.

Reply:
Jup, I actually got it working with my brew-installed ssh-agent without any changes.

Reply:
Ok wait, scratch that. I think it's still starting the macOS agent and not the brew installed one...